### ISSUE TITLE
Error after upgrade to 1.5.0: strict mode breaks 'src' caller info

### ISSUE THREAD

**Issue by platy** [trentm/node-bunyan#296]
I just upgraded to 1.5.0 from 1.4.0 - pretty quick, I know.

I get a failure with this exception:
```
/home/mike/vertex/node_modules/bunyan/lib/bunyan.js:156
    Error.captureStackTrace(this, getCaller3Info);
         ^
TypeError: Object.defineProperty called on non-object
   at defineProperty (native)
   at getCaller3Info (/home/mike/vertex/node_modules/bunyan/lib/bunyan.js:156:11)
   at mkRecord (/home/mike/vertex/node_modules/bunyan/lib/bunyan.js:939:27)
```

I can dig deeper - I just thought I'd report first.

**Comment by Ognian** [comment]
+1

**Comment by Ognian** [comment]
This is a side effect of
```'use strict';```
a dirty workaround is to comment it out...

**Comment by MuncleUscles** [comment]
+1 Just ran into the same issue, downgrading back to 1.4 for now

**Comment by trentm** [comment]
Hrm. Sorry about that. I'll take a look tonight.

Obviously a test case for this is lacking.

**Comment by timelessvirtues** [comment]
Same issue, pinning to version 1.4.0 for now.

**Comment by alex94puchades** [comment]
Same issue here

**Comment by trentm** [comment]
I'm going to remove `'use strict';` to get `src:true` working again. I started #297 to look at re-adding strict mode to bunyan.js later without breaking things.

**Comment by trentm** [comment]
Should be fixed in v1.5.1. There is also a guard in `getCaller3Info()` now so that if bunyan.js *is* in strict mode it at least won't throw.
